internal/controller/cloudstorage_controller.go

```
@@ -155,14 +197,38 @@ func (b CloudStorageReconciler) Reconcile(ctx context.Context, req ctrl.Request)
+			},
+		)
+		if statusErr := b.Client.Status().Update(ctx, &bucket); statusErr != nil {
+			logger.Error(statusErr, "Failed to update CloudStorage status after bucket existence check failure")
+		}
 		return ctrl.Result{RequeueAfter: 1 * time.Minute}, nil
 	}
 
 	// Update status with updated value
 	bucket.Status.LastSynced = &metav1.Time{Time: time.Now()}
 	bucket.Status.Name = bucket.Spec.Name
+	
```

golangci-lint (2.2.2)

[error] File is not properly formatted

(gci)